client/components/jetpack/masterbar/index.jsx

```diff
@@ -57,6 +58,7 @@ const JetpackCloudMasterBar = () => {
 			>
 				<JetpackLogo size={ 28 } full={ ! isNarrow || isExteriorPage } />
 			</Item>
+			<PortalNav />
```

Comment:
This seems to have inflated the main entry point by over 1%. Could we consider `AsyncLoad` it @atanas-dev?